episodes/31-multi-files.md

```diff
@@ -66,14 +66,14 @@ includes all the filenames.
 
 Here is our updated program.
 
-<pre>
+```python
 import sys
 import pandas
 # we need to import part of matplotlib
 # because we are no longer in a notebook
 import matplotlib.pyplot as plt
 
-<b>for filename in sys.argv[1:]:</b>
+for filename in sys.argv[1:]:
```

Comment:
We should still try to do something to draw people's attention to this particular line.

Reply:
```suggestion
for filename in sys.argv[1:]: # <= this is the line that is changing
```